streamlit_folium/frontend/src/index.tsx

```diff
@@ -111,6 +113,18 @@ function onDraw(e: any) {
   return onLayerClick(e)
 }
 
+function removeLayer(e: any) {
+  const global_data = window.__GLOBAL_DATA__
+  global_data.selected_layers.delete(e.layer["wmsParams"]["layers"]);
```

Comment:
Does this generically apply to any type of map layer added? Or is this just certain types of layers?